react/features/mobile/share-meeting/components/ShareMeetingInfoButton.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+
+import _ from 'lodash';
+import { Share } from 'react-native';
+import { connect } from 'react-redux';
+
+import { getInviteURL } from '../../../base/connection';
+import { translate } from '../../../base/i18n';
+import { AbstractButton } from '../../../base/toolbox';
+import type { AbstractButtonProps } from '../../../base/toolbox';
+import { updateDialInNumbers } from '../../../invite';
+
+type Props = AbstractButtonProps & {
+
+    /**
+     * The object representing the dialIn feature.
+     */
+    _dialIn: Object,
+
+    /**
+     * The current URL of the conference.
+     */
+    _inviteURL: string,
+
+    /**
+     * URL where the current conference is being live-streamed.
+     */
+    _liveStreamViewURL: ?string,
+
+    /**
+     * The redux {@code dispatch} function.
+     */
+    dispatch: Function,
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function
+}
+
+/**
+ * An implementation of a button for showing the {@code ShareMenu}.
+ */
+class ShareMeetingInfoButton extends AbstractButton<Props, *> {
+    accessibilityLabel = 'Share meeting info';
+    iconName = 'icon-info';
+    label = 'info.tooltip';
+
+    /**
+     * Update the dial-in numbers, if needed.
+     *
+     * @inheritdoc
+     */
+    componentDidMount() {
+        this.props._dialIn.numbers
+            || this.props.dispatch(updateDialInNumbers());
+    }
+
+    /**
+     * Creates a message describing how to dial into the conference. Include
+     * the following items, in order.
+     *
+     * - URL.
+     * - LiveStream URL.
+     * - Phone numbers.
+     *
+     * TODO: look into sharing this with InfoDialog on web.
```

Comment:
I left this here because it looks like a larger change than I originally thought to share this with web (due to its reliance on data from the store and the translate function). I also thought of creating a function that got all of those as parameters... WDYT?